.gitignore

```diff
@@ -1,20 +1,281 @@
-.DS_Store
-.mypy_cache
+# Suggestions for gitignore - Python
+# https://github.com/github/gitignore/blob/master/Python.gitignore
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```

Comment:
for this one, I don't think we should copy-paste code without understanding it, so happy to add ignores to something specific you need, but we don't have any C extensions, so the code is vestigial